exasol_script_languages_container_ci_setup/lib/run_generate_buildspec.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+from pathlib import Path
+from typing import Tuple, Optional
+import jsonschema
+
+import pkg_resources
+
+
+from exasol_script_languages_container_ci_setup.lib import render_template
+
+
+class Flavor(object):
+    """"
+    Holds the name and the formatted name used for generating the buildspec.
+    """
+
+    def __init__(self, flavor: str):
+        self.flavor_original = flavor
+        self.flavor_formatted = flavor.replace(".", "").replace("-", "_")
+
+    def __str__(self):
+        return f"Flavor(original={self.flavor_original}, formatted={self.flavor_formatted})"
+
+    def __repr__(self):
+        return self.__str__()
+
+
+def validate_config_file(config_file: Optional[str]):
+    if config_file is not None:
+        with open(config_file, "r") as config_file_:
+            config = json.load(config_file_)
+            config_schema = json.loads(render_template("config_schema.json"))
+            jsonschema.validate(instance=config, schema=config_schema)
+            ignored_folders = config["build_ignore"]["ignored_folders"]
```

Comment:
are these only folders or also files?

Reply:
Hm, good point. Actually it can be files, too. I will change the key.